bundle/run.sh

```diff
@@ -820,47 +865,6 @@ while true; do
   sleep 1
 done
 
-# certificate-authority
```

Comment:
must be started before grpc-gw is?

Reply:
Yes. Because grpc connects to certificate-authority